common_features.mk

```diff
@@ -503,6 +503,10 @@ ifeq ($(strip $(SPLIT_KEYBOARD)), yes)
     POST_CONFIG_H += $(QUANTUM_DIR)/split_common/post_config.h
     OPT_DEFS += -DSPLIT_KEYBOARD
 
+    ifeq ($(strip $(SYNC_STATUS_LEDS)), yes)
+        OPT_DEFS += -DSYNC_STATUS_LEDS
+    endif
+
```

Comment:
Would prefer that this was just a simple define, as we have with `split_mods_enable`